apps/core/static/cosmos/css/about.css

```diff
@@ -44,4 +44,11 @@
 
 .partners .carousel-item {
     justify-content: center;
-}
\ No newline at end of file
+}
+
+span.carousel-control-next-icon {
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='%23ff0000' width='8' height='8' viewBox='0 0 8 8'%3e%3cpath d='M2.75 0l-1.5 1.5L3.75 4l-2.5 2.5L2.75 8l4-4-4-4z'/%3e%3c/svg%3e");
+}
+
+span.carousel-control-prev-icon {
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='%23ff0000' width='8' height='8' viewBox='0 0 8 8'%3e%3cpath d='M5.25 0l-4 4 4 4 1.5-1.5L4.25 4l2.5-2.5L5.25 0z'/%3e%3c/svg%3e");
```

Comment:
Is there a reason for changing these icons? They now are kinda obnoxiously red, and always visible even when not needed. Controls were never really intended for this carousel, as the idea was to have it automatically cycle through them if there is not enough space to fit all of them.

Reply:
Ah, the red wasn't showing for me so I forgot to remove these lines
Why no controls though? If someone with a short attention span is scrolling through and only sees one sponsor they miss the other one(s), while with controls it's clear that there are multiple and will still work even if there is only one

Reply:
I guess that it makes sense for mobile yeah, not sure if it is needed for desktop though, as that should show 3 at the same time. But it is fine to add it either way, just less obtrusive as it is not the main thing we want to focus attention to.